src/components/Calculator.js

```diff
@@ -1,34 +1,58 @@
+/* eslint-disable react/destructuring-assignment */
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Kindly re-enable and fix the linters because they are like a standard or quality assurance that should be maintained throughout the project.